docker-for-windows/index.md

```diff
@@ -707,6 +707,63 @@ Check out these [Docker Cloud topics](/docker-cloud/index.md) to learn more:
 Need a direct link to Cloud? [Take me to Docker
 Cloud](https://cloud.docker.com/){: target="_blank" class="_" }.
 
+### Kubernetes
+
+**Kubernetes is only available in Docker for Windows 18.02 CE Edge users who
```

Comment:
Same as above, this is available for Edge users, no beta program & no need to log in.